Dockerfile

```diff
@@ -5,11 +5,12 @@ RUN apt-get update && \
     apt-get install -y build-essential && \
     apt-get install -y lsb-release && \
     apt-get install -y wget && \
+    apt-get install -y s-nail && \
```

Comment:
If we can configure the second container in the cronjob to copy and send the logs, you can remove `s-nail` from the patsy image.

Reply:
Okay, I'm almost finished with reconfiguring my cronjobs, so once I get that working I can just edit the dockerfile and remove s-nail